channels/ru.m3u

```diff
@@ -467,8 +467,6 @@ http://live-saha.cdnvideo.ru/saha/saha/playlist.m3u8
 https://live-vestinn.cdnvideo.ru/vestinn/nn24-khl/playlist.m3u8
 #EXTINF:-1 tvg-id="NikaTV.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="https://nikatv.ru/public/img/logo2.png" group-title="",Ника ТВ (576p) [Not 24/7]
 https://live-nikatv.cdnvideo.ru/nikatv/nikatv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="NLOHD.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",НЛО HD (720p)
-http://spacetv.in/stream/DIPK8H19CP8/344.m3u8
```

Comment:
https://xx001.vivat.live/t-0301.0f6f.d/sd/00047/2m/index.m3u8

Ukrainian channel

Reply:
Added in #5500